### ISSUE TITLE
SSL/TLS error in LiveOptionChainProvider

### ISSUE THREAD

**Issue by StefanoRaggi** [QuantConnect/Lean#1952]
#### Expected Behavior
No error should be thrown when calling the `GetOptionContractList` method.

#### Actual Behavior
When calling `GetOptionContractList` method this error is thrown:
- Mono:
```System.Net.WebException: Error: SecureChannelFailure (The authentication or decryption has failed.)```
- Windows:
```System.Net.WebException : The request was aborted: Could not create SSL/TLS secure channel.```

#### Potential Solution
The `LiveOptionChainProvider` fetches the option chain information from the OCC website.
Apparently, it seems they upgraded the security protocol in their web server and our web request is failing, at least `TLS 1.1` is now required and NET 4.5.2 does not have it enabled by default.

#### Reproducing the Problem
This unit test is now failing on current master branch:
```
        [Test]
        public void LiveOptionChainProviderReturnsData()
        {
            var provider = new LiveOptionChainProvider();
            var result = provider.GetOptionContractList(Symbols.AAPL, DateTime.Today);

            Assert.IsTrue(result.Any());
        }
```

#### System Information
NET 4.5.2 and Mono in QC cloud

#### Checklist
- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)
- [x] I have provided detailed steps to reproduce the issue

**Comment by StefanoRaggi** [comment]
Reopening this issue because the problem persists in the QC cloud (Mono) and needs further investigation.

This test confirms that the OCC web server only supports TLS 1.1 and TLS 1.2:
https://www.ssllabs.com/ssltest/analyze.html?d=www.theocc.com

**Comment by StefanoRaggi** [comment]
Closed by #1968
